sdk/download.go

```diff
@@ -269,11 +269,12 @@ func UpdateFile(file, url string, client *http.Client) error {
 	}
 	t.Close()
 	tempFile := t.Name()
-	defer os.Remove(tempFile)
+	os.Remove(tempFile) // let DownloadFile create it and use umask
```

Comment:
I've just changed it to set the mode to something useful.  It's still kind of ugly since I don't see a way to get the umask without changing it.  If that's an issue, I can just move the mode change to the command instead of in the `sdk` library so nothing else gets affected by this.